pkg/alertmanager/statefulset.go

```diff
@@ -267,6 +267,10 @@ func makeStatefulSetSpec(logger log.Logger, a *monitoringv1.Alertmanager, config
 		amArgs = append(amArgs, "--web.external-url="+a.Spec.ExternalURL)
 	}
 
+	if len(a.Spec.EnableFeatures) > 0 {
```

Comment:
we need a check on a.Spec.Version too since it's only supported with 0.27.0 and onwards.

Reply:
You can see an example of how this can be done in this PR: https://github.com/prometheus-operator/prometheus-operator/pull/6326